proposed/2022/DotnetListPackageMachineReadableJsonOutput.md

```diff
@@ -772,6 +772,139 @@ Outputs json for format version 1, if it's not specified then latest version'll
 }
 ```
 
+### Error handling
+
+In case of an error it would be written into `problems` section. But in case there is parameter, options error then it may defaults back to console output instead of json output.
+Also in case of runtime there's error then it may not show json output at all because it needs all the data for json output before display.
+
```

Comment:
Two questions:

- Is JSON output printed in STDOUT or STDERR?
- Can we add a non-zero process exit code when an error occurs?

Reply:
>> Is JSON output printed in STDOUT or STDERR?

See Andy's [response](https://github.com/NuGet/Home/pull/11983#issuecomment-1196228788).

>> Can we add a non-zero process exit code when an error occurs?

Please check latest changes. Yes, we'll return non-zero exit code. For current implementation it would be general error code `1`.